allennlp/data/dataset_readers/reading_comprehension/dqa.py

```diff
@@ -0,0 +1,127 @@
+import json
+import logging
+from typing import Any, Dict, List, Tuple
+
+from overrides import overrides
+
+from allennlp.common.file_utils import cached_path
+from allennlp.data.dataset_readers.dataset_reader import DatasetReader
+from allennlp.data.instance import Instance
+from allennlp.data.dataset_readers.reading_comprehension import util
+from allennlp.data.token_indexers import SingleIdTokenIndexer, TokenIndexer
+from allennlp.data.tokenizers import Token, Tokenizer, WordTokenizer
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+@DatasetReader.register("dqa")
+class DQAReader(DatasetReader):
+    """
+    Reads a JSON-formatted QUAC file and returns a ``Dataset`` where the ``Instances`` have four
```

Comment:
Because this is a new dataset, it would be helpful to give the unabbreviated name and link to the paper.